packages/terra-application-header-layout/tests/wdio/application-header-spec.js

```diff
@@ -38,4 +38,14 @@ describe('Application Header Layout', () => {
 
     Terra.should.matchScreenshot('Not Visible', { viewports, selector: '#test-header' });
   });
+
+  describe('Displays a application header layout with logo and utilities only', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/terra-application-header-layout/application-header-layout/application-header-with-logo-and-utilities-only');
+      browser.waitForVisible('#test-header');
+    });
+
+    Terra.should.matchScreenshot({ viewports, selector: '#test-header' });
```

Comment:
Should use the new `Terra.should.validateElement()` instead of `Terra.should.matchScreenshot()` and `Terra.should.beAccessible()`

Reply:
Done: https://github.com/cerner/terra-framework/pull/604/commits/03196e2e93e12b1783514fcedd11a309475db86a.